CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 2.32.2.20240516 Linux re-build
+
+*This release has the same Fluent Bit contents as 2.32.0, and is simply a linux-only re-build for recent patches in dependencies installed in the image. There are no windows images for this release.* 
+* Amazon Linux Base: [2.0.20240503.0](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-20240508.html)
+
+
```

Comment:
Would you remove the extra indentation, for consistency with the rest of the change log?